etc/config/hook.defaults.properties

```diff
@@ -0,0 +1,20 @@
+
+interpreted=true
+supportsBinary=false
+supportsExecute=true
+versionFlag=--version
+needsMulti=false
+
+compilerType=hook
+compilers=&hook
+defaultCompiler=hook010
+
+group.hook.compilers=hook010
```

Comment:
We faced some issues with having the .defaults compilers ids share them with the .amazon one. Can you add a `def` prefix to this ids?

Reply:
Sorry. I did not understand. Did you say to replace `hook010` with `defhook010` in that `hook.defaults.properties` file?

Reply:
I noticed that python uses a suffix, so I'll do something like this: `hook010def`. Looks good?

Reply:
Perfect! Yes